src/nxdoc/nuxeo-server/administration/opensearch-setup.md

```diff
@@ -0,0 +1,963 @@
+        version: '5'
+    -
+        author: Benoit Delbosc
+        date: '2014-11-10 14:34'
+        message: ''
+        version: '4'
+    -
+        author: Benoit Delbosc
+        date: '2014-11-10 14:32'
+        message: ''
+        version: '3'
+    -
+        author: Benoit Delbosc
+        date: '2014-11-10 14:13'
+        message: ''
+        version: '2'
+    -
+        author: Benoit Delbosc
+        date: '2014-11-10 12:34'
+        message: ''
+        version: '1'
+---
+
+{{! excerpt}}
+This page provides several configuration use cases for OpenSearch.
+{{! /excerpt}}
+
+## Setting up an OpenSearch Cluster
+
+### OpenSearch Supported Version
+
+The Nuxeo Platform communicates with OpenSearch using the [HTTPS Rest protocol](https://opensearch.org/docs/1.3/clients/java-rest-high-level/) (port 9200 by default), which provides looser coupling with OpenSearch.
+
+{{#> callout type='warning' }}
+The OpenSearch Java high-level REST client will be deprecated starting with OpenSearch version 3.0.0 and will be removed in a future release. OpenSearch recommends switching to the Java client instead.
+{{/callout}}
+
+OpenSearch compatibility details can be found on the [Compatibility Matrix]({{page page='compatibility-matrix#opensearch'}}) page.
+
+The default configuration uses an embedded OpenSearch instance that runs in the same JVM as the Nuxeo Platform's.
+
+{{#> callout type='warning' }}
+This embedded mode **is only for testing purpose** and should not be used in production, neither OpenSearch nor Nuxeo can support an embedded installation.
+{{/callout}}
+
+For production you need to setup an OpenSearch cluster.
+
+### Installing OpenSearch
+
+Refer to the [OpenSearch documentation](https://opensearch.org/docs/1.3/install-and-configure/install-opensearch/index/) to install OpenSearch. Basically:
+
+- [Don&rsquo;t use network file system for node storage.](https://opensearch.org/docs/1.3/install-and-configure/install-opensearch/index/#file-system-recommendations)
+- [Make sure the right ports are open.](https://opensearch.org/docs/1.3/install-and-configure/install-opensearch/index/#network-requirements)
+
+
+#### Recommended Tuning
+
+If you have a large number of documents or if you use Nuxeo in cluster you may reach the default configuration limitation, here are some [recommended tuning OpenSearch options](https://opensearch.org/docs/1.3/install-and-configure/install-opensearch/index/#important-settings)
+
+This page shows you how to prevent the heap to be swapped. Swapping can dramatically decrease performance and stability, so you should ensure it is disabled on production clusters.
+
+It also gives guidance regarding the JVM configuration values like below, and how to avoid duplication and unexpected behaviors:
+```
+OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m
+```
+
+To reduce disk IO you should consider changing the default [translog](https://www.elastic.co/guide/en/elasticsearch/reference/6.3/index-modules-translog.html)
```

Comment:
This is a Nuxeo configuration not opensearch configuration, 
this page should contain only stuff specific to opensearch (i.e. different from elastic)

Reply:
The translog ?
I removed the page anyway. So it's only on the ES page